sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/download/http/HttpConnectionCheckTask.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.appcenter.distribute.download.http;
+
+import android.os.AsyncTask;
+
+import com.microsoft.appcenter.utils.storage.SharedPreferencesManager;
+
+import java.io.File;
+
+import static com.microsoft.appcenter.distribute.DistributeConstants.PREFERENCE_KEY_DOWNLOADED_RELEASE_FILE;
+
+public class HttpConnectionCheckTask extends AsyncTask<Void, Void, Void> {
```

Comment:
Can this be package-private? I am confused because the constructor is and there is no factory.

Reply:
yes, it can